test/prefer-default-parameters.js

```diff
@@ -1,13 +1,5 @@
-import test from 'ava';
```

Comment:
> 100644 → 100755

Don't change the mode of the file. See the UI above here.

Reply:
Sorry.